clio-server/src/main/scala/org/broadinstitute/clio/server/webservice/JsonWebService.scala

```diff
@@ -8,4 +12,13 @@ import org.broadinstitute.clio.util.json.ModelAutoDerivation
   */
 trait JsonWebService
     extends ModelAutoDerivation
-    with ErrorAccumulatingCirceSupport
+    with ErrorAccumulatingCirceSupport {
+
+  final implicit val jsonStreamingSupport: JsonEntityStreamingSupport =
```

Comment:
Maybe a comment here would be helpful, why it's needed and what operations will use this.

Reply:
Added.